src/styles/style.css

```diff
@@ -0,0 +1,117 @@
+.body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  box-sizing: border-box;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__subtitle {
+  box-sizing: border-box;
+  color: #616070;
+  margin: 0 0 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__response {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  margin: 0 0 16px;
+}
+
+.card__response .stars {
+  display: flex;
+  height: 16px;
+}
+
+.card__response .stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.card__response .stars__star:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
+}
+
+.card__text {
+  color: #060b35;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__price {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 18px;
+  margin-bottom: 16px;
+}
+
+.card__price .left {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__buy {
+  text-decoration: none;
+  background-color: #00acdc;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #fff;
+  height: 40px;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+}
+
+.card__buy:hover {
+  border: 1px solid #00acdc;
+  color: #00acdc;
```

Comment:
For consistency with the SCSS file, consider using the color value defined in the `$color-button` variable instead of the hardcoded `#00acdc`.